lib/taskana-core/src/main/java/pro/taskana/common/internal/util/WorkingDaysToDaysConverter.java

```diff
@@ -8,14 +8,13 @@
 import java.time.LocalDate;
```

Comment:
why are we refactoring this class?

Reply:
I made this sneaky peaky. You saw nothing 👍